CorsixTH/Lua/humanoid_actions/seek_reception.lua

```diff
@@ -73,7 +73,7 @@ local function action_seek_reception_start(action, humanoid)
     local x = best_desk.tile_x + orientation.use_position[1]
     local y = best_desk.tile_y + orientation.use_position[2]
     local dist = humanoid.world:getPathDistance(humanoid.tile_x, humanoid.tile_y, x, y)
-    humanoid:updateDynamicInfo(_S.dynamic_info.patient.actions.on_my_way_to
+    humanoid:setDynamicInfoText(_S.dynamic_info.patient.actions.on_my_way_to
```

Comment:
This one threw an error after re-testing the game but it only happened once... Haven't been able to replicate

```bash
An error has occurred!
Almost anything can be the cause, but the detailed information below can help the developers find the source of the error.
Running: The timer handler.
A stack trace is included below, and the handler has been disconnected.
Lua\humanoid_actions\seek_reception.lua:76: method 'setDynamicInfoText' is not callable (a nil value)
stack traceback:
        Lua\humanoid_actions\seek_reception.lua:76: in field '?'
        Lua\entities\humanoid.lua:476: in method 'startAction'
        Lua\entities\humanoid.lua:577: in local 'timer_function'
        Lua\entity.lua:216: in method 'tick'
        Lua\world.lua:1085: in method 'onTick'
        Lua\app.lua:1125: in function <Lua\app.lua:1122>
        (...tail calls...)
```

Reply:
Found the issue. Fixing...